mesa_geo/geospace.py

```diff
@@ -203,24 +222,80 @@ def _recreate_rtree(self, new_agents=None):
         self.idx = index.Index(index_data)
         self.idx.agents = {id(agent): agent for agent in agents}
 
-    def update_bbox(self, bbox=None):
-        """Update bounding box of the GeoSpace."""
-        if bbox:
-            self.bbox = bbox
-        elif not self.agents:
-            self.bbox = None
+    def add_agents(self, agents):
+        """Add a list of GeoAgents to the layer without checking their crs.
+
+        GeoAgents must have the same crs to avoid incorrect spatial indexing results.
```

Comment:
I would additionally tell people to read `GeoSpace._check_agent` for how to do it, but this is fine I suppose.

Reply:
Sure I can add that.